lib/game_stats.rb

```diff
@@ -1,4 +1,94 @@
 
 
+
+
 module GameStats
+
+	def highest_total_score(season) # integer
+		if away_winners(season) > home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) < home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def lowest_total_score(season)
+		if away_winners(season) < home_winners(season)
+			away_winners(season)
+		elsif away_winners(season) > home_winners(season)
+			home_winners(season)
+		end
+	end
+
+	def get_games(season) # helper
+		@all_games = @games
+	end
+
+	def home_winners(season) # helper
+		@home_wins = []	
+			@games.each do |game|
```

Comment:
definitely a nit picky thing, but I don't think you need to tab in here - only when you're starting a new code block like under a def, it-do block, conditional or enumerator

Reply:
Good call, that was an accidental extra tab. Will fix this when I go back to fix stuff today!